lux/extensions/ui/css/navbar.py

```diff
@@ -15,67 +15,100 @@ def add_css(all):
     # STYLING
     navbar = vars.navbar
     #
+    navbar.transition.duration_in = '0.6s'
+    navbar.transition.duration_out = '0.2s'
+    navbar.transition.duration = '0.8s'
+    navbar.collapse.background = '#eaeaea'
+    navbar.collapse.max_height = px(999)
+    navbar.collapse.margin.top = px(2)
+    #
     # Navbar Height
     navbar.height = px(60)
+    navbar.big_height = px(80)
     navbar.small_height = px(60)
+    navbar.offset = px(1)
     navbar.lineheight = px(20)
-    navbar.transition.duration = '0.8s'
     navbar.padding = 0.5*(navbar.height-navbar.lineheight)
     navbar.small_padding = 0.5*(navbar.small_height-navbar.lineheight)
     vars.animate.fade.top = navbar.height
     navbar.divider.color = '#ddd'
     #
-    trans = navbar.transition
     collapse_width = px(cfg['NAVBAR_COLLAPSE_WIDTH'])
 
     css('.navbar.navbar-static-top',
+        css(' .navbar-nav',
+            css('> li > a',
+                line_height=navbar.lineheight)),
         css(' .navbar-brand',
             css(' img', height=navbar.small_height),
             padding=0),
-        min_height=navbar.small_height)
+        height=navbar.small_height)
 
     css('.navbar.navbar-static-top',
         css(' .navbar-nav',
             css('> li > a',
                 line_height=navbar.lineheight)))
 
+    css('.navbar',
+        border_top='none')
```

Comment:
I used this, because between navbar and opened menu was 1px line gap (this was due to current theme).